src/components/DocumentStatus/IssuedBy.tsx

```
@@ -27,6 +28,18 @@ const getV2FormattedDomainNames = (verificationStatus: VerificationFragment[]) =
 
   const formatIdentifier = (fragment: VerificationFragmentWithData<VerificationFragmentData[]>): string | undefined => {
     switch (fragment.name) {
+      case "OpencertsRegistryVerifier":
+        const opencertsFragment = getOpencertsRegistryVerifierFragment(verificationStatus);
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)